packages/tasks/src/model-libraries.ts

```diff
@@ -185,7 +185,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "deepforest",
 		docsUrl: "https://deepforest.readthedocs.io/en/latest/",
 		repoUrl: "https://github.com/weecology/DeepForest",
-		countDownloads: `path_extension:"pt" OR path_extension:"pl"`,
+		countDownloads: `path_extension:".safetensors" OR path_extension:".json"`,
```

Comment:
```suggestion
```

According to what you explained, I believe that using the default (which tracks `config.json`) would suffice.